packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -189,13 +192,16 @@ const CFDsListing = observer(() => {
                 <React.Fragment>
                     {combined_cfd_mt5_accounts.map((existing_account, index: number) => {
                         const list_size = combined_cfd_mt5_accounts.length;
+                        const account_disabled = !existing_account?.status?.includes('enabled');
                         const has_mt5_account_status =
-                            existing_account?.status || is_idv_revoked
+                            existing_account?.status ||
+                            is_idv_revoked ||
+                            (existing_account.action_type === 'multi-action' && account_disabled)
```

Comment:
Let's request a proper `disabled` status from the API to avoid random checks.

Reply:
hi @matin-deriv , according to Kean Fei the `enabled` flag is directly from mql and they would need to do some mapping to return the proper `disabled` status. But due to code practice, BE prefers to keep the status as it is from metaquotes hence returning the `enabled`

should we still block this PR and insist on the flag...?